Authentication/AuthenticationBootstrapper.swift

```diff
@@ -12,8 +12,9 @@ import Foundation
     Bootstrapper to start the application.
     Takes care of starting the authentication process before the main View Controller of the app when necessary,
     and after the user logs out.
+    The authentication process includes login, signup and recover password logic.
 */
-public class AuthenticationBootstraper<User: UserType, SessionService: SessionServiceType where SessionService.User == User> {
+public class AuthenticationBootstrapper<User: UserType, SessionService: SessionServiceType where SessionService.User == User> {
```

Comment:
In this case I think it makes more sense to have a property. Because this will be the most common way to configure the view and by exposing a property we avoid having to inherit from the bootstrapper to customize how the view will look like.